webpack.config.js

```diff
@@ -6,6 +6,7 @@ var index = require('./lib/index-static.jsx');
 
 module.exports = {
   entry: './lib/main.jsx',
+  devtool: 'source-map',
```

Comment:
Hmm, what do you mean by "work for Firefox"? In general I've found that Firefox has very poor support for source maps--they work fine in the debugger, but not the console. 

For instance, here's what a `debugger` statement does in Firefox, with the code in this PR applied:

![2015-03-05_12-11-36](https://cloud.githubusercontent.com/assets/124687/6510000/1e2267d2-c331-11e4-8049-f87d979b8aeb.jpg)

As you can see there it's correctly mapping to the original JSX source.

However, see that `console.log("HI")` right before the `debugger` statement? This is what that looks like in the console:

![2015-03-05_12-12-29](https://cloud.githubusercontent.com/assets/124687/6509980/ffd8d658-c330-11e4-804f-8eab3947c702.jpg)

This is particularly frustrating for me because I spend much more of my time in the console than I do in the debugger. But I've never found a project that works any differently on Firefox, so when I really need source-mapped `console.log()` statements, I just switch to Chrome. It's kind of Chrome devtools' killer feature for me. :(